features/FEATURE_UVISOR/includes/uvisor/api/inc/uvisor-lib.h

```diff
@@ -17,6 +17,8 @@
 #ifndef __UVISOR_API_UVISOR_LIB_H__
 #define __UVISOR_API_UVISOR_LIB_H__
 
+#include "includes/uvisor-lib/uvisor-lib.h"
```

Comment:
Actually when I used your `cmsis/core_cmSecureAccess.h` change I was able to remove the include! Though I just tried the binary and it doesn't seem to work. I'm afraid my knowledge is limited on what files need to include what, maybe this needs to be looked at more in depth?